cpp/rrtmgp_conversion.h

```diff
@@ -435,6 +445,23 @@ typename KView::non_const_value_type maxval(const KView& view)
   return rv;
 }
 
+// Get min val of View
+template <typename KView>
+typename KView::non_const_value_type minval(const KView& view)
+{
+  using scalar_t    = typename KView::non_const_value_type;
+  using exe_space_t = typename KView::execution_space;
+
+  scalar_t rv;
+  Kokkos::parallel_reduce(
+    Kokkos::RangePolicy<exe_space_t>(0, view.size()),
+    KOKKOS_LAMBDA(size_t i, scalar_t& lmax) {
+      const scalar_t val = view.data()[i];
+      if (val > lmax) lmax = val;
```

Comment:
This is confusing; is this correct? It seems like you would want to do something like `if (val < lmin) lmin = val` here.

Reply:
Nice catch! Copy/paste error from maxval.